UI/src/view/point_asset/pointAssetForm.js

```diff
@@ -311,7 +315,7 @@ root.PointAssetForm = function() {
     return '' +
         '    <div class="form-group editable form-point-asset">' +
         '      <label class="control-label">' + property.localizedName + '</label>' +
-        '      <p class="form-control-static">' + (propertyValues[propertyDefaultValue].propertyDisplayValue || '-') + '</p>' +
+        '      <p class="form-control-static">' + (_.find(propertyValues, ['propertyValue', propertyValue]).propertyDisplayValue || '-') + '</p>' +
```

Comment:
Shouldn't be validate _.find(propertyValues, ['propertyValue', propertyValue]) is empty before do the .propertyDisplayValue ?